rca/programmes/models.py

```diff
@@ -515,6 +480,28 @@ class ProgrammePage(BasePage):
 
     search_fields = BasePage.search_fields + []
 
+    def get_alumni_stories(self):
+        # TODO, once we can query by programme slug, we'll need to
+        # add the slug as prefix to the cache key
+        if not cache.get("programme_latest_alumni_stories"):
+            cache.set(
+                "programme_latest_alumni_stories",
+                content.pull_alumni_stories(),
+                settings.API_CONTENT_CACHE,
```

Comment:
`API_CONTENT_CACHE` is this supposed to be the timeout? Could you name this setting something more intuitive please?